fuel_cells/intelligent_energy_fcm_801/manifest.yml

```diff
@@ -79,6 +91,52 @@ telemetry:
     unit: joule
     display_name: Total Run Energy
 
+  fault_flags_a:
+    type: integer
+    display_name: Fault Flags A
+  fault_flags_b:
+    type: integer
+    display_name: Fault Flags B
+  fault_flags_c:
+    type: integer
+    display_name: Fault Flags C
+  fault_flags_d:
+    type: integer
+    display_name: Fault Flags D
+  total_can_packets:
+    type: integer
+    display_name: Total CAN Packets Received
+  messages_0x400:
+    type: string
+    display_name: CAN Messages 0x400
+    description: Troubleshooting logs for analysis by Intelligent Energy service team
```

Comment:
@nikitug is this metric visible on Dashboard without enum values?

Reply:
Nope, we do not need it on the dashboard, nobody actually can read it without IE. The only use case is to get this data from API and send it to IE for analysis.